charts/gardener-extension-provider-openstack/values.yaml

```diff
@@ -66,7 +66,7 @@ config:
     storage:
       className: gardener.cloud-fast
       capacity: 25Gi
-      provisioner: kubernetes.io/cinder
+      provisioner: cinder.csi.openstack.org
```

Comment:
As far as I see this StorageClass is used for `etcd-main` PVs on the Seed. So the surprising thing for me was that the out-of-tree volume provisioners are compatible with the in-tree ones and for example can provision a PV that use a StorageClass with the old in-tree provisioner (I assume this is coming from the csi-translation magic). Which means that the out-of-tree driver can handle both `cinder.csi.openstack.org` and `kubernetes.io/cinder`.
So if do this change, I think it will break Seeds that don't support the `cinder.csi.openstack.org` provisioner. Also the `provisioner` field of StorageClass is immutable. So I assume the update will fail with failure.

So bottom line: we still would like to preserve compatibility with Seeds that only support `kubernetes.io/cinder`.

/hold

@rfranzke , any thoughts?

Reply:
+1 @ialidzhikov, I guess it's not needed to change this (it's anyways just an example value that can be overridden).